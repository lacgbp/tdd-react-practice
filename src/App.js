import React from 'react';
import { Provider } from 'react-redux';
import ToDo from './components/ToDo';
import store from './redux/store';

function App() {
  return (
    <Provider store={store}>
      <ToDo />
    </Provider>
  );
}

export default App;
