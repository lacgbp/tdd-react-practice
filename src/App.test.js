import React from 'react';
import ReactDOM from 'react-dom';

import { shallow } from 'enzyme';

import App from './App';

describe('Jest deafult test', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});

describe('Enzyme default test', () => {
  it('render snapshot', () => {
    const snapShot = shallow(<App />);
    expect(snapShot).toMatchSnapshot();
  });
})
