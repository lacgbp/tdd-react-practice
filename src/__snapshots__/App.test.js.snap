// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Enzyme default test render snapshot 1`] = `<ToDo />`;
